pkg/process/logging.go

```diff
@@ -11,20 +11,29 @@ import (
 	"github.com/zeebo/errs"
 	"go.uber.org/zap"
 	"go.uber.org/zap/zapcore"
+
+	"storj.io/storj/pkg/cfgstruct"
 )
 
 var (
 	// Error is a process error class
 	Error = errs.Class("process error")
 
-	logLevel    = zap.LevelFlag("log.level", zapcore.WarnLevel, "the minimum log level to log")
-	logDev      = flag.Bool("log.development", false, "if true, set logging to development mode")
-	logCaller   = flag.Bool("log.caller", false, "if true, log function filename and line number")
-	logStack    = flag.Bool("log.stack", false, "if true, log stack traces")
+	logLevel = zap.LevelFlag("log.level", func() zapcore.Level {
+		if isDev() {
+			return zapcore.InfoLevel
+		}
+		return zapcore.DebugLevel
```

Comment:
Wait, was it supposed to be this way?

For development INFO, and production DEBUG?

Reply:
nope i got it backwards, thank you

Reply:
well, the devops team *is* enabling debug in production, but you're right, i meant the opposite